File: GrowthWare_VB/GrowthWareFramework/ModelObjects/MClientChoices.vb
Namespace ModelObjects
    ''' <summary>
    ''' All propertyies represent the column names in the ZF_ACCT_CHOICES table.
    ''' </summary>
    ''' <remarks></remarks>
	<Serializable(), CLSCompliant(True)> _
	Public NotInheritable Class MClientChoices
		Private Sub New()

		End Sub

		Shared ReadOnly Property RecordsPerPage() As String
			Get
				Return "RECORDS_PER_PAGE"
			End Get
		End Property

		Shared ReadOnly Property AnonymousClientChoicesState() As String
			Get
				Return "AnonymousClientChoicesState"
			End Get
		End Property

		Shared ReadOnly Property SessionName() As String
			Get
				Return "ClientChoicesState"
			End Get
		End Property

		Shared ReadOnly Property AccountName() As String
			Get
				Return "ACCT"
			End Get
		End Property

		Shared ReadOnly Property SecurityEntityID() As String
			Get
				Return "SE_SEQ_ID"
			End Get
		End Property

		Shared ReadOnly Property SecurityEntityName() As String
			Get
				Return "SE_NAME"
			End Get
		End Property

		Shared ReadOnly Property BackColor() As String
			Get
				Return "BACK_COLOR"
			End Get
		End Property

		Shared ReadOnly Property LeftColor() As String
			Get
				Return "LEFT_COLOR"
			End Get
		End Property

		Shared ReadOnly Property HeadColor() As String
			Get
				Return "HEAD_COLOR"
			End Get
		End Property

		Shared ReadOnly Property SubheadColor() As String
			Get
				Return "SUB_HEAD_COLOR"
			End Get
		End Property

		Shared ReadOnly Property ColorScheme() As String
			Get
				Return "COLOR_SCHEME"
			End Get
		End Property

		Shared ReadOnly Property Action() As String
			Get
				Return "FAVORIATE_ACTION"
			End Get
		End Property
	End Class
End Namespace

File: GrowthWare_CS/GrowthWareFrameworkModel/Profiles/MClientChoices.cs
using System;

namespace GrowthWare.Framework.Model.Profiles
{
	/// <summary>
	/// All propertyies represent the column names in the ZF_ACCT_CHOICES table.
	/// </summary>
	/// <remarks></remarks>
	[Serializable(), CLSCompliant(true)]
	public sealed class MClientChoices
	{

		private MClientChoices()
		{
		}

		public static string RecordsPerPage {
			get { return "RECORDS_PER_PAGE"; }
		}

		public static string AnonymousClientChoicesState {
			get { return "AnonymousClientChoicesState"; }
		}

		public static string SessionName {
			get { return "ClientChoicesState"; }
		}

		public static string AccountName {
			get { return "ACCT"; }
		}

		public static string SecurityEntityID {
			get { return "SE_SEQ_ID"; }
		}

		public static string SecurityEntityName {
			get { return "SE_NAME"; }
		}

		public static string BackColor {
			get { return "BACK_COLOR"; }
		}

		public static string LeftColor {
			get { return "LEFT_COLOR"; }
		}

		public static string HeadColor {
			get { return "HEAD_COLOR"; }
		}

		public static string SubheadColor {
			get { return "SUB_HEAD_COLOR"; }
		}

		public static string ColorScheme {
			get { return "COLOR_SCHEME"; }
		}

		public static string Action {
			get { return "FAVORIATE_ACTION"; }
		}
	}
}
